ts/packages/node/src/stores/TomlFile.ts

```diff
@@ -0,0 +1,80 @@
+import { ConfigStore, ConfigStoreQuery, Config, Convert } from '@configu/ts';
```

Comment:
Please make sure to run `npm run lint` to make sure the code is linted according to the linting rules of the project.